### PR TITLE
Add minor features to the guppy

### PR DESCRIPTION
**Pull request by NieKam**
Because it was very hard and annoying to simulate shake event on emulator, I've added label to the main activity that triggers shake event listener.

Change preview:
Screenshot_1571216922
request_list

I also added request time and status code  to the requests list item, because subtitle was almost the same as title of the request.

Better to review commit by commit.

### CODE REVIEW

File: guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package com.jnj.guppy.interceptor
+
+
+class HttpStatus {
```
Comment (CCorrado) [inline guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt:18]: is this class needed? For instance, HttpURLConnection has most of the commonly used ones built in. https://developer.android.com/reference/java/net/HttpURLConnection.html#HTTP_BAD_REQUEST

File: guppy/src/test/java/com/jnj/guppy/GuppyInterceptorTests.kt [hunk 2]
```diff
@@ -34,6 +35,7 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mock
 import org.mockito.junit.MockitoJUnitRunner
+import kotlin.math.log
```
Comment (CCorrado) [inline guppy/src/test/java/com/jnj/guppy/GuppyInterceptorTests.kt]: is this being used?

**Comment by CCorrado** [comment]
Additionally, some of the added code doesn't map to unit tests. Might want to add a few more, otherwise I love the UI improvements!

```
------------------ Code Coverage Failed -----------------------
- branch coverage rate is: 50.0%, minimum is 53.0%
- complexity coverage rate is: 54.94%, minimum is 56.0%
- method coverage rate is: 78.89%, minimum is 80.0%
---------------------------------------------------------------
```

File: guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt (hunk 1 above)
Comment (NieKam) [reply guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt:18]: It's needed to map status code (int) to message. For instance 201 **Created**.

File: guppy/src/androidTest/java/com/jnj/guppy/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -1,11 +1,13 @@
 package com.jnj.guppy;
 
 import android.content.Context;
-import androidx.test.InstrumentationRegistry;
```
Comment (NieKam) [inline guppy/src/androidTest/java/com/jnj/guppy/ExampleInstrumentedTest.java:4]: I run optimize imports in android studio. I missed that I touched example file.

Comment (CCorrado) [reply guppy/src/androidTest/java/com/jnj/guppy/ExampleInstrumentedTest.java:4]: All good!

File: guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt (hunk 1 above)
Comment (CCorrado) [reply guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt:18]: So I don't see the need for Guppy to be responsible for defining all the Http Status/Response codes, i see that this was once part of the Apache library but deprecated in API 22: https://android.googlesource.com/platform/external/apache-http/+/refs/heads/lollipop-mr1-release/src/org/apache/http/HttpStatus.java

If you need to map status codes, use the constants available in `HttpURLConnection` but I don't see the need to have the readable strings defined in Guppy.

**Review by CCorrado — APPROVED** [review]

File: guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt (hunk 1 above)
Comment (CCorrado) [reply guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt:18]: Resolving as the iOS team implemented a similar method.

**Comment by NieKam** [comment]
oh, I wanted to add new PR but I forgot that this PR is still open, so commits were added here. I reverted both.

**Pull request merged**